sdk/identity/azure-identity/azure/identity/aio/_credentials/chained.py

```diff
@@ -22,6 +23,15 @@ class ChainedTokenCredential(SyncChainedTokenCredential):
     :type credentials: :class:`azure.core.credentials.TokenCredential`
     """
 
+    async def __aenter__(self):
```

Comment:
Why redefine an empty `__aenter__`? You already inherit it from the `AsyncCredentialBase`, no?

Reply:
Never mind - the `__aenter__` is marked as abstract in the base class. Which it probably shouldn't be.